src/components/widgets/accounts-cash-projection/accounts-cash-projection.directive.coffee

```diff
@@ -33,6 +33,17 @@ module.controller('WidgetAccountsCashProjectionCtrl', ($scope, $q, $filter, $tim
   $scope.chartId = ->
     "cashProjectionChart-#{w.id}"
 
+  # == Setup Highchart Options ============================================================================
+
+  _highChartOptions = Object.freeze
+    chartType: 'line'
+    chartOnClickCallbacks: []
+    currency: w.metadata.currency
```

Comment:
As the variable `_highChartsOptions` is being defined in the controller function level of this directive, it will be evaluation as soon as the function is invoked internally by angular. This means, I am quite confident, w.metadata will be undefined at that exact point in time. 

This is essentially being decared as a constant, but as it has dynamic data inside it, it needs to either be `$onInit` (still fakey with asyc data), or better `$onChanges`, or even better (less code), re-evaluated when `w.format` is called.

Reply:
Good catch. Will put back into w.format.